tests/spatial/rotation_test.py

```diff
@@ -103,3 +103,13 @@ def test_can_set_value_of_0d_variable():
     var = rotation(value=value)
     var.value += value
     assert np.array_equal(var.value, value + value)
+
+
+def test_inverse():
+    rng = np.random.default_rng()
+    value = rng.random((4, ))
```

Comment:
I am not sure this is even well defined. This is not a normalized quaternion, and as far as I remember scipp does not do that automatically.

Reply:
Indeed. We do not document that quaternions must be normalized. Adding it to the docs.